org.mwc.cmap.legacy/src/MWC/GUI/Layers.java

```diff
@@ -488,7 +488,12 @@ private static Layer checkLayer(final Layer layer, final String name)
         {
           return l;
         }
-        else if ( !(l instanceof ICantContainLayers) )

```

Comment:
@saulhidalgoaular  - I've slightly restructured, so I can document why we have that path through the logic

It won't be a problem, will it?

Reply:
Your change is perfect!!!!